include/sys/dnode.h

```diff
@@ -463,6 +463,11 @@ typedef struct dnode_stats {
 	 * Number of times multiple threads attempted to allocate a dnode
 	 * from the same block of free dnodes.
 	 */
+	kstat_named_t dnode_alloc_next_chunk;
+	/*
+	 * Number of times dmu_object_alloc*() reached the end of the existing
+	 * object ID chunk and advanced to a new one.
+	 */
```

Comment:
The comment should be above the `kstat_named_t`.

Reply:
fixed